docs/_conversion/maintaining.md

```diff
@@ -0,0 +1,140 @@
+
+Locating the current integration
+
+On your “My Integrations” page, make sure you’re working on the copy of your integration in the “Integrations” section - not the “Integrations Built with Legacy Web Builder” section.
+
+![](https://cdn.zappy.app/50e1b6f3dfe56965ffce80cc7ad879cd.png)
+
+## Versions
+
+The most immediate and important difference you’ll notice is that the new platform introduces the concept of [versions](https://platform.zapier.com/docs/versions).  
+
+In the legacy Web Builder, you’d make a clone of your project to make changes to your integration. In the new platform the workflow is similar, but you can manage multiple version of your app for a lot more control and flexibility.  
+
+The top level of your integration is a thin container that houses the title, logo, description of your app.  All partnership metrics and analytics will reference this container.
+
+Inside this container are versions.  Versions hold all the actual workings of your integration - triggers, actions, authentication configuration, etc.  You are free to make big changes from one version to another.  
+
+## New management features: promotion, migration, deprecation
+
+_To make a change to your integration._
+
+- Create a new version by cloning an existing version in the versions section found in the left hand navigation bar.
+- Make your changes to the new version.  
+- If you have invited testers they can create Zaps with this new version before anyone else has access to it.
+- When you’re ready for the world to use your new version, promote it.  Promoting a version makes it the version new users will see when they make Zaps. 
+- If you didn’t make breaking changes, migrate your users to the new version.  Unlike Web Builder, this doesn’t happen automatically. You can control when it happens.  A recommended practice is to migrate percentages of your users and watch for problems in your logs before migrating every one.  If you made breaking changes, you can simply leave users using the older version, or deprecate the old version to force users to reconfigure their Zaps to accommodate your changes.
+
+## About breaking changes
+
+A breaking change happens when you need to change something that prevents us from being able to switch users Zaps to the new version without their interaction.  Breaking changes include:
+
+- Changing authentication
+- Adding a new required field
+- Removing a trigger/action/search
+
+
+## Deprecation
+
+Deprecation is an extremely useful feature introduced in the new platform.  If you made a breaking change and can’t migrate users to your new version, you might choose to deprecate the older version.  This will automatically email users of the older version to let them know they need to revisit the Zap Editor and reconfigure their Zaps for your newer version.  On the deprecation date you set any remaining Zaps still using the deprecated version will be paused.
```

Comment:
> you might choose to deprecate the older version

this is true, but I'd frame it as "hopefully, all your versions will continue to work indefinitely". It's ok if users are on older versions. They should be able to stay there as long as they don't need new features or bug fixes introduced in later versions.